js/main.js

```diff
@@ -65,3 +66,25 @@ let createTodoElement = (elementId, todoText) => {
   let list = document.getElementById("todoList");
   list.appendChild(todo);
 }
+
+
+let showTime = () => {
+  let date = new Date();
+  let h = date.getHours(); // 0 - 23
+  let m = date.getMinutes(); // 0 - 59
+  let s = date.getSeconds(); // 0 - 59
+
+  
+  h = (h < 10) ? "0" + h : h;
+  m = (m < 10) ? "0" + m : m;
+  s = (s < 10) ? "0" + s : s;
+  
+  let time = h + ":" + m + ":" + s;
+  document.getElementById("clock").innerText = time;
+  document.getElementById("clock").textContent = time;
+  
+  setTimeout(showTime,0);
```

Comment:
Do we really want to call it all the time? According to [mozzilla](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout#Reasons_for_delays_longer_than_specified) this will try to make the call every 4ms when the tab is active and 1000ms when not active. I think it would be okay to just make it try to update it every 1000ms (1s)

Reply:
Okay, why I set it to zero was because it would be one second behind the real clock, but if it updates 1000ms when not in focus it will just get outdated whenever the user switches tab

But then again, this will only show when they open a new tab, then they will open whatever site they want

So the question is, consistently behind with 1 second or on time if they dont switch tabs?

Reply:
This is not a problem anymore when we hide the seconds right? But as you said, maybe it won't be a problem to call the update all the time because this page will most probably only be on the screen for a short period of time. So if you want I'm okay with it to be set to 0